src/http/api/views/account.posts.view.ts

```diff
@@ -286,6 +289,118 @@ export class AccountPostsView {
         };
     }
 
+    async getPostsFromOutbox(
+        account: Account,
+        contextAccountId: number,
+        limit: number,
+        cursor: string | null,
+    ): Promise<Result<AccountPosts, GetPostsError>> {
+        if (!account.isInternal) {
```

Comment:
Currently this will never be the case as this function is only called for internal accounts but I added this check so that if this function is used elsewhere, in the future, it gives an error while reading for external accounts.